test/terraform/provider-pinning.bats

```diff
@@ -12,22 +16,22 @@ function teardown() {
   unset TF_CLI_ARGS_init
 }
 
-@test "check if terraform-config-inspect is installed" {
```

Comment:
Why are we deleting this?

Reply:
It is not a test of the PR/code, it is a test of the test harness. It should not be called out as and `ok` or `not ok` test of the PR.